src/environments/environment.ts

```diff
@@ -14,4 +14,4 @@ export const environment = {
  * This import should be commented out in production mode because it will have a negative impact
  * on performance if an error is thrown.
  */
-// import 'zone.js/dist/zone-error';  // Included with Angular CLI.
+// import 'zone.js/plugins/zone-error';  // Included with Angular CLI.
```

Comment:
I see this change here but no change in zone.js dependency. Is this right?

Reply:
Yeah, answered above.

Reply:
above you answered that zone.js doesn't need to be upgraded - but I think anand is asking: then why would this change? was it previously wrong?

It looks like the zone docs agree that it was:
https://github.com/angular/angular/tree/master/packages/zone.js#bundles